cg/services/orders/store_order_services/store_microbial_order.py

```diff
@@ -1,170 +1,91 @@
-        status_data = self.order_to_status(order)
-        self._fill_in_sample_ids(samples=status_data["samples"], lims_map=lims_map)
+        project_data, lims_map = self.lims.process_lims(
+            samples=order.samples,
+            customer=order.customer,
+            ticket=order._generated_ticket_id,
+            order_name=order.name,
+            workflow=ORDER_TYPE_WORKFLOW_MAP[order.order_type],
+            delivery_type=DataDelivery(order.delivery_type),
+        )
+        self._fill_in_sample_ids(samples=order.samples, lims_map=lims_map)
 
         # submit samples to Status
-        samples = self.store_items_in_status(
-            customer_id=status_data["customer"],
-            order=status_data["order"],
-            ordered=project_data["date"] if project_data else datetime.now(),
-            ticket_id=order.ticket,
-            items=status_data["samples"],
-            comment=status_data["comment"],
-            data_analysis=Workflow(status_data["data_analysis"]),
-            data_delivery=DataDelivery(status_data["data_delivery"]),
-        )
+        samples = self.store_order_data_in_status_db(order)
         return {"project": project_data, "records": samples}
 
-    @staticmethod
-    def order_to_status(order: OrderIn) -> dict:
-        """Convert order input for microbial samples."""
-
-        status_data = {
-            "customer": order.customer,
-            "order": order.name,
-            "comment": order.comment,
-            "data_analysis": order.samples[0].data_analysis,
-            "data_delivery": order.samples[0].data_delivery,
-            "samples": [
-                {
-                    "application": sample.application,
-                    "comment": sample.comment,
-                    "control": sample.control,
-                    "name": sample.name,
-                    "organism_id": sample.organism,
-                    "priority": sample.priority,
-                    "reference_genome": sample.reference_genome,
-                    "volume": sample.volume,
-                }
-                for sample in order.samples
-            ],
-        }
-        return status_data
-
-    def store_items_in_status(
-        self,
-        comment: str,
-        customer_id: str,
-        data_analysis: Workflow,
-        data_delivery: DataDelivery,
-        order: str,
-        ordered: datetime,
-        items: list[dict],
-        ticket_id: str,
-    ) -> [Sample]:
+    def store_order_data_in_status_db(self, order: MicrobialOrder) -> list[DbSample]:
         """Store microbial samples in the status database."""
 
-        sample_objs = []
-
-        customer: Customer = self.status.get_customer_by_internal_id(
-            customer_internal_id=customer_id
-        )
+        customer: Customer = self.status.get_customer_by_internal_id(order.customer)
         new_samples = []
-        status_db_order = Order(
+        db_order = DbOrder(
```

Comment:
I have been using the CRUD function `self.status_db.add_order` for instantiating Orders instead of using the class constructor